app/controllers/Application.scala

```diff
@@ -23,6 +23,14 @@ class Application(
   import actionRefiners._
 
   implicit val ar = assets
+
+  private def applyCircles(queryParam: String, id: String, js: String, modifiedId: String, modifiedJs: String): (String, String) = {
```

Comment:
I think this method's name needs to be more specific because, on line 55/63, there is a check where `newDesigns` can equal `"circles"`, in which case the behaviour is different from when it is equal to `"circles-garnett"`.